### ISSUE TITLE
Cannot run `dbt-osmosis==0.10.3` on `dbt-core>=1.3,<1.4`"

### ISSUE THREAD

**Issue by waligob** [z3z1ma/dbt-osmosis#48]
I created and activated a fresh virtual environment with `dbt-osmosis==0.10.3` and `dbt-core>=1.3,<1.4`(currently resolves to `dbt-core==1.3.3`) with the following command:
```console
pyenv virtualenv 3.9.7 dbt-osmosis-test && pyenv activate $_ && pip install "dbt-osmosis==0.10.3" "dbt-core>=1.3,<1.4"
```

However, upon running various `dbt-osmosis` command (e.g. `dbt-osmosis --version`, `dbt-osmosis --help`), I get the following error:

```
dbt-osmosis --help
Traceback (most recent call last):
  File "/Users/waligob/.pyenv/versions/dbt-osmosis-test/bin/dbt-osmosis", line 5, in <module>
    from dbt_osmosis.main import cli
  File "/Users/waligob/.pyenv/versions/3.9.7/envs/dbt-osmosis-test/lib/python3.9/site-packages/dbt_osmosis/main.py", line 17, in <module>
    from dbt_osmosis.core.macros import inject_macros
  File "/Users/waligob/.pyenv/versions/3.9.7/envs/dbt-osmosis-test/lib/python3.9/site-packages/dbt_osmosis/core/macros.py", line 4, in <module>
    from dbt_osmosis.core.osmosis import DbtProject
  File "/Users/waligob/.pyenv/versions/3.9.7/envs/dbt-osmosis-test/lib/python3.9/site-packages/dbt_osmosis/core/osmosis.py", line 22, in <module>
    from dbt.contracts.graph.parsed import ColumnInfo, ManifestNode  # type: ignore
ImportError: cannot import name 'ManifestNode' from 'dbt.contracts.graph.parsed' (/Users/waligob/.pyenv/versions/3.9.7/envs/dbt-osmosis-test/lib/python3.9/site-packages/dbt/contracts/graph/parsed.py)
```

**Comment by z3z1ma** [comment]
@waligob 

This has been addressed in `dbt-osmosis==0.10.5`

Will close the issue once confirmed on your end, thanks 😄

**Comment by z3z1ma** [comment]
I have confirmed it runs on the mentioned dbt version as of latest release on pypi!
`pip install --upgrade dbt-osmosis`
